src/test/resources/bad-extension.yml

```diff
@@ -0,0 +1 @@
+extensionName: hello-world
```

Comment:
This name doesn't match the name implied by the filename.  Can we have "bad" in the name? ;-)

Reply:
haha sure!